### ISSUE TITLE
fixOnSav does not work on source.js.embedded.html scope

### ISSUE THREAD

**Issue by jackolo** [AtomLinter/linter-eslint#992]
## Issue Type
Bug

## Issue Description
fixOnSave does not work for vue files, which have source.js.embedded.html scope.
Running "Linter Eslint: Fix File" command manually on vue file works.
fixOnSave works on regular js files.

## Linter-eslint debugging information
```
Atom version: 1.19.4
linter-eslint version: 8.2.1
ESLint version: 3.19.0
Hours since last Atom restart: 0.6
Platform: darwin
Using local project ESLint from: /Users/jacek/Code/JS/cell/node_modules/eslint
Current file's scopes: [
  "text.html.vue",
  "source.js.embedded.html",
  "meta.delimiter.comma.js"
]
linter-eslint configuration: {
  "fixOnSave": true,
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDir": "",
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": []
}
```

**Comment by Arcanemagus** [comment]
From https://github.com/eslint/eslint/issues/7510#issuecomment-275947091 it sounds like `eslint-plugin-html` was doing _Bad Things™️_ to make this work before. On the plus side? A proper way of doing this was implemented in https://github.com/eslint/eslint/pull/9090 which released in ESLint v4.7.0. I'm guessing the author of `eslint-plugin-html` removed their hacky method when that released thus "breaking" this for you.

tl;dr: Update to ESLint v4.7.0 or higher 😉.
